src/calculateRentalCost.js

```diff
@@ -3,8 +3,25 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const dayRentCost = 40;
+
+  const longTerm = 7;
+  const shortTerm = 3;
+
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
```

Comment:
The constants 'longTerm', 'shortTerm', 'longTermDiscount', and 'shortTermDiscount' are descriptive and help to avoid magic numbers in the code. This is a good example of using constants to make the code more readable and maintainable.